python/python/tests/test_transaction.py

```diff
@@ -550,7 +558,15 @@ async def test_standalone_transaction(self, redis_client: RedisClient):
         assert isinstance(result[0], str)
         assert "# Memory" in result[0]
         assert result[1:4] == [OK, OK, value]
-        assert result[4:11] == [2, 2, 2, ["Bob", "Alice"], 2, OK, None]
+        assert result[4:11] == [
```

Comment:
len of the response array depends on the redis version

Reply:
I didn't add sort_ro in here, only in the transaction, which the response is added only in case of valkey higher than 7.